unitTestSupportFunctions/calculateLift.m

```diff
@@ -0,0 +1,30 @@
+%% Function documentation
+%
+% Calculate lift force on a single body with no-slip boundary conditions in a rectangular domain
+%
+%               Input :
+%           FComplete : The complete force vector
+%              fldMsh : Nodes and elements for the fluid mesh
+%              homDBC : The global numbering of the DOFs where homogeneous
+%                       Dirichlet boundary conditions are applied
+%          parameters : Flow parameters
+%
+%
+%              Output :
+%                  Fy : Force (lift) on the body in y-direction
+%
+%% Function main body
+function Fy = calculateLift (FComplete, fldMsh, homDBC, parameters)
```

Comment:
Add another variable i.e. noDimensions to account for the dimensionality of the problem. For the 2D test cases this variable needs to be set equal to 3 (ux, uy, uz) whereas for 3D cases it should be set to 4.

Reply:
I am not quite sure what do you mean by "noDimensions", since both functions calculateLiftForce and calculateDragForce only calculate force Fx and Fy espectively. If you want a to handle 3D problems we would need another function that sums the entries in z-direction, or rewrite the function in style [Fx, Fy, Fz] = calculateForcesOnBody.

Reply:
I mean that within this function, you are hard-coding a multiplication and division by 3. This works only in the case where there are 3 DOFs per node as per the 2D NS equations. However, I would like to have this general, meaning that the user can also use this function for the 3D case in which this hard-coded multiplication/division by 3 will have to replaced by 4. Therefore, I recommend to have an additional input variable noDimensions which for this case can be set to 3 and used within the function. 

Can you please consult the comments I made within my revision instead of this random ones that I made accidentally? It will be easier to go ahead with reviewing. 

Thanks!

Reply:
Ok, I will think about the best way to approact it, since I will most likely also have to change the two other functions that prescribe parabolic inlet and find nodes on the body of interest, they currently only work in 2D